### DISCUSSION TITLE
Why can't I use https in Github Page? 为什么我无法使用HTTPS在Github Page？

### DISCUSSION THREAD

**Discussion by zuanshijia** [community/community#119391] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

English:
first,I have changed my DNS resolution,and it's official.
![求助截图2](https://github.com/community/community/assets/101307794/5cd74f1b-704f-4219-8619-faecfee51fa4)
My websize can only use HTTP
That's Why？
![求助截图1](https://github.com/community/community/assets/101307794/78742db5-99c7-469a-8000-9c6cfbaaa409)
My domain name is fine, then I checked my certificate, nothing wrong.
I wish someone could explain it to me, thank you.
Chinese:首先，我已经更改了我的DNS解析，且是官方要求的
但是我的网站仍只能用HTTP
这是为什么
我的域名无问题，我检查了我的网站证书，都无问题
有大佬能给我解释一下吗？谢谢！
By zuanshijia

**Comment by yoannchaudet** [comment]
Hello,

Since you are using Cloudflare to front your domain, GitHub Pages cannot issue a TLS certificate on your behalf.

You will have to figure out the option (it exists) on Cloudflare's side to enable HTTPS only.
